plugin/build_baron.go

```diff
@@ -59,9 +62,18 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 			return errors.Errorf(`Failed validating configuration for project "%s": `+
 				"bf_suggestion_timeout_secs must be zero when bf_suggestion_url is blank", projName)
 		}
-		if proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != "" {
+		// the webhook cannot be used if the default build baron creation and search is configurd
+		if webhookConfigured && len(proj.TicketCreateProject) != 0 {
```

Comment:
These two if statements do the same thing but have different messages, right? That's kind of confusing, can you just do it once, but add a field like `"ticket_create_project_length": proj.TicketCreateProject` to the logged error?